app/views/templates/_form.html.erb

```diff
@@ -84,6 +83,70 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="help">
+      <div>
+        <%= _('Provisioning templates use ERB syntax') %>
```

Comment:
Is this meant to be a heading? If it's meant to be part of the following sentence then it's missing a full stop, though doesn't make much sense as a full sentence.